### PR TITLE
added my file

### PR DESCRIPTION
**Pull request by crystal-b**

### CODE REVIEW

File: crystal-b.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+CRYSTALTRON
```
Comment (afeld) [inline crystal-b.md:1]: Could really use better documentation on this line.

**Comment by OhJia** [comment]
Hi Crystal!

**Comment by dennyabrain** [comment]
I disagree

**Comment by SongHia** [comment]
that's crazy talk

**Comment by Craigson** [comment]
this is not a comment.

**Comment by afeld** [comment]
Looks good :+1:

**Pull request merged**